coalib/output/Logging.py

```diff
@@ -1,28 +1,80 @@
 from datetime import datetime
-import json
 import io
+import json
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
I did not understand this comment :sweat: 
The `LOG_LEVEL` is a dict used by me here to convert log levels into strings 
for example:
`logging.DEBUG` which is just a constant for `int(10)` is converted into `DEBUG`